platform/pulp/plugin/changeset/model.py

```diff
@@ -0,0 +1,182 @@
+from gettext import gettext as _
+from logging import getLogger
+
+from django.db.utils import IntegrityError
+from django.db import transaction
+from django.core.files import File
+
+
+log = getLogger(__name__)
+
+
+class Remote:
+    """
+    Represents content related things contained within the remote repository but
+    is not contained within the local (pulp) repository.
+
+    Attributes:
+        model (Model): A remote (wanted) model object.
+    """
+
+    def __init__(self, model):
+        """
+        Args:
+            model (Model): A remote (wanted) model object.
+        """
+        self.model = model
+
+    def save(self):
+        """
+        Save the model.
+        """
+        self.model.save()
+        log.debug(_('Model: %(m)s - saved'), {'m': self})
+
+
+class RemoteContent(Remote):
+    """
+    Represents content contained within the remote repository but is not
+    contained within the local (pulp) repository.
+
+    Attributes:
+        model (pulp.plugin.Content): A real content model object.
```

Comment:
No, it is the `Content` model instance.  I'm new to django but feel sure I've seen (perhaps incorrectly) both the model class and instance referred to as "model".  And, I could be mistaken.  I mainly wanted to make it clear that the attribute is a content model object associated with the remote content object.  I think this supports more readable code everywhere else `content.model` is used.  The alternative of naming it something like `content` would lead to wonky and confusing semantics like `content.content` or `remote_content.content` when used.

In most places in the pulp code base, I strongly support naming the model instances for what they are instead of just `model` has you suggest.